spec/models/form/sales/questions/person_working_situation_spec.rb

```diff
@@ -44,259 +44,127 @@
-      end
-
-      it "has expected check answers card number" do
-        expect(question.check_answers_card_number).to eq(2)
-      end
+  context "when person 2" do
+    let(:question_id) { "ecstat2" }
+    let(:person_index) { 2 }
 
-      it "has the correct inferred_check_answers_value" do
-        expect(question.inferred_check_answers_value).to eq([
-          { "condition" => { "ecstat2" => 10 }, "value" => "Prefers not to say" },
-        ])
-      end
+    before do
+      allow(page).to receive(:id).and_return("person_2_working_situation")
     end
 
-    context "and joint purchase" do
-      let(:person_index) { 3 }
-      let(:question_id) { "ecstat3" }
-
-      before do
-        allow(page).to receive(:id).and_return("person_1_working_situation_joint_purchase")
-      end
-
-      it "has the correct id" do
-        expect(question.id).to eq("ecstat3")
-      end
-
-      it "has the correct header" do
-        expect(question.header).to eq("Which of these best describes Person 1’s working situation?")
-      end
-
-      it "has the correct check_answer_label" do
-        expect(question.check_answer_label).to eq("Person 1’s working situation")
-      end
-
-      it "has expected check answers card number" do
-        expect(question.check_answers_card_number).to eq(3)
-      end
+    it "has the correct id" do
+      expect(question.id).to eq("ecstat2")
+    end
 
-      it "has the correct inferred_check_answers_value" do
-        expect(question.inferred_check_answers_value).to eq([
-          { "condition" => { "ecstat3" => 10 }, "value" => "Prefers not to say" },
-        ])
-      end
+    it "has the correct header" do
+      expect(question.header).to eq("Which of these best describes Person 2’s working situation?")
     end
-  end
 
-  context "when person 2" do
-    context "and not joint purchase" do
-      let(:question_id) { "ecstat3" }
-      let(:person_index) { 3 }
-
-      before do
-        allow(page).to receive(:id).and_return("person_2_working_situation")
-      end
-
-      it "has the correct id" do
-        expect(question.id).to eq("ecstat3")
-      end
-
-      it "has the correct header" do
-        expect(question.header).to eq("Which of these best describes Person 2’s working situation?")
-      end
-
-      it "has the correct check_answer_label" do
-        expect(question.check_answer_label).to eq("Person 2’s working situation")
-      end
-
-      it "has expected check answers card number" do
-        expect(question.check_answers_card_number).to eq(3)
-      end
-
-      it "has the correct inferred_check_answers_value" do
-        expect(question.inferred_check_answers_value).to eq([
-          { "condition" => { "ecstat3" => 10 }, "value" => "Prefers not to say" },
-        ])
-      end
+    it "has the correct check_answer_label" do
+      expect(question.check_answer_label).to eq("Person 2’s working situation")
```

Comment:
Again like in `spec/models/form/sales/questions/person_gender_identity_spec.rb`, we probably want to keep this but remove the duplicate test further up this file?